calicoctl/commands/resources.go

```diff
@@ -182,6 +182,10 @@ type commandResults struct {
 
 	// The results returned from each invocation
 	resources []unversioned.Resource
+
+	// The Calico API client used for the requests (useful if required
+	// again).
```

Comment:
Just that there is no real reason to return the client in the response data ... but in this case it is useful to have access to it so that we don't have to create another client.